lib/sinatra/respond_with.rb

```diff
@@ -1,7 +1,7 @@
 require 'sinatra/json'
 require 'sinatra/base'
 
-$KCODE = "UTF-8"
+$KCODE = "UTF-8" unless RUBY_VERSION > "1.9.0"
```

Comment:
Between that and https://github.com/sinatra/sinatra-contrib/pull/169#issuecomment-110176938 :+1: